cli/pkg/output/printer_factory.go

```diff
@@ -59,16 +63,30 @@ func Stack2Console(ctx context.Context, stack stackservice.StackInfo) StackConso
 		uniqueMap[endpoint] = true
 	}
 	for endpoint := range uniqueMap {
+		// filter out the k8s cluster endpoints
+		if strings.Contains(endpoint, "svc.cluster") {
+			continue
+		}
 		endpoints = append(endpoints, endpoint)
 	}
 
+	abbrevRepo := strings.TrimPrefix(strings.TrimSuffix(stack.Repo, ".git"), "git@github.com:")
+	abbrevOwner := strings.TrimSuffix(stack.Owner, "@chanzuckerberg.com")
```

Comment:
We need to make this configurable in the future

Reply:
yea